### ISSUE TITLE
Wrong wrapping of links-at-bottom in Markdown

### ISSUE THREAD

**Issue by hoenic07** [stkb/Rewrap#63]
In Markdown links could be added to the bottom of the page, so that they don't clutter the actual document. This looks like this:
```
[Link1]: http://www.link1.com/lorem_ipsum_lorem_ipsum_lorem_ipsum
[Link2]: http://www.link2.com/lorem_ipsum_lorem_ipsum_lorem_ipsum
```
Unfortunately this will be rewrapped to:
```
[Link1]: http://www.link1.com/lorem_ipsum_lorem_ipsum_lorem_ipsum [Link2]:
http://www.link2.com/lorem_ipsum_lorem_ipsum_lorem_ipsum
```

In my opinion the expected behavior is that one link should be in one line (like in the first code snipped)

**Comment by stkb** [comment]
I'll look in to supporting this. In the mean time you should be able to use the workaround of adding 2 spaces to the end of each line; this will preserve the line break.

**Comment by alphang** [comment]
> In the mean time you should be able to use the workaround of adding 2 spaces to the end of each line; this will preserve the line break.

Hi - this workaround is _super_ useful. Is it documented somewhere? 

I was trying to hard wrap a list without getting rid of the line breaks, and it took me a long time to find this post.

**Comment by stkb** [comment]
[This will be fixed in the next version](https://github.com/stkb/Rewrap/releases/tag/beta).

**Comment by stkb** [comment]
Fixed in v1.16.0
